Description: Revert 3ddb16bd5bb7146d125a7d68e4e9f3b54c381a20
 This reverts the mentioned commit, since it depends on a newer
 version of libwayland, that is not yet available on noble
Author: Constantin Piber <cp.piber@gmail.com>

Index: source/CMakeLists.txt
===================================================================
--- source.orig/CMakeLists.txt
+++ source/CMakeLists.txt
@@ -129,7 +129,7 @@ pkg_check_modules(
   IMPORTED_TARGET
   xkbcommon
   uuid
-  wayland-server>=1.22.90
+  wayland-server
   wayland-protocols>=1.43
   cairo
   pango
Index: source/src/Compositor.cpp
===================================================================
--- source.orig/src/Compositor.cpp
+++ source/src/Compositor.cpp
@@ -311,7 +311,7 @@ void CCompositor::initServer(std::string
         Debug::m_trace = true;
 
     // set the buffer size to 1MB to avoid disconnects due to an app hanging for a short while
-    wl_display_set_default_max_buffer_size(m_wlDisplay, 1_MB);
+    // wl_display_set_default_max_buffer_size(m_wlDisplay, 1_MB);
 
     Aquamarine::SBackendOptions options{};
     options.logFunction = aqLog;
